tests/conftest.py

```diff
@@ -0,0 +1,18 @@
+import pytest
```

Comment:
I've not used pytest yet but it does add a dependency (at least in Python 2.7 this isn't part of the stdlib), whereas previously there were no external deps. If we do this could you add a requirements.txt with pytest? Or could we just use base unittest for this bit?